src/libsync/connectionvalidator.cpp

```diff
@@ -235,6 +237,13 @@ void ConnectionValidator::slotCapabilitiesRecieved(const QVariantMap &json)
     auto caps = json.value("ocs").toMap().value("data").toMap().value("capabilities");
     qDebug() << "Server capabilities" << caps;
     _account->setCapabilities(caps.toMap());
+
+    QString serverVersion = caps.toMap()["core"].toMap()["status"].toMap()["version"].toString();
+    if (!serverVersion.isEmpty()) {
+        _account->setServerVersion(serverVersion);
+        qDebug() << "Version from capabilities instead of status.php:" << serverVersion;
```

Comment:
You can try reading the capabilities, but to most people it will look like 

![](https://i.ytimg.com/vi/kqUR3KtWbTk/maxresdefault.jpg)